### DISCUSSION TITLE
how to addition 1 column values i need [Jun 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2596] [Discord Threads]
> 🙋‍♂️ **sidheequesdq**  
## how to addition 1 column values?  i need add title 14 values to title15. its possible?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/987448974879391805/Screenshot_2022-06-17_at_11.37.52_PM.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **sidheequesdq**  
#### Anyone?  
#  
> 💁 **o1lab**  
#### adding values from two different rows within another column is not possible  
#  
> 🙋‍♂️ **sidheequesdq**  
#### any other idea for this?  
#  
> 🙋‍♂️ **sidheequesdq**  
#### i need to create a reward program for every transaction  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987448975718244432)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
